src/CovidAppointmentTable.js

```diff
@@ -39,10 +42,20 @@ export function sortAndFilterData(
     { sortKey, sortAsc },
     onlyShowAvailable
 ) {
-    const filteredData = onlyShowAvailable
-        ? data.filter((entry) => entry.hasAppointments)
-        : data;
-    const newData = filteredData.sort((a, b) => {
+    // Filter the locations that have "non-stale" data
+    const oldestGoodTimestamp = new Date() - (tooStaleMinutes * 60 * 1000);
+    let filteredData = data.filter(
+        (entry) => (new Date(entry.timestamp).getTime() === 0
```

Comment:
I think it would be clearer to check for null/falsy values here, instead of relying on new Date(null) to be time=0. So
```
({timestamp}) => !timestamp || timestamp >= oldestGoodTimestamp
```

Reply:
(switching out `(entry)` for `({timestamp})` because all we care about is `entry.timestamp` but if you prefer to be a little more verbose then I'm cool with keeping `(entry)`)